openslides/motions/serializers.py

```diff
@@ -241,9 +245,9 @@ def update(self, motion, validated_data):
                 setattr(motion, key, validated_data[key])
 
         # Workflow.
-        workflow = validated_data.get('workflow')
-        if workflow is not None and workflow != motion.workflow:
-            motion.reset_state(workflow)
+        workflow_id = validated_data.get('workflow_id')
+        if workflow_id is not None and workflow_id != motion.workflow:
```

Comment:
motion.workflow does not return a workflow object, but a workflow-id. See by todo above.